libnetwork/store.go

```diff
@@ -6,12 +6,10 @@ import (
 
 	"github.com/docker/docker/libnetwork/datastore"
 	"github.com/docker/libkv/store/boltdb"
-	"github.com/docker/libkv/store/etcd"
 	"github.com/sirupsen/logrus"
 )
 
 func registerKVStores() {
-	etcd.Register()
 	boltdb.Register()
```

Comment:
Also need to check if this store is actually used (I thought it was, because I know libnetwork uses boltdb for some things), but not sure if this code is actually used for that.